cmd/fossil/client/client.go

```diff
@@ -14,7 +14,8 @@ import (
 	"strings"
 	"time"
 
-	"github.com/dburkart/fossil/api"
+	"github.com/chzyer/readline"
+	fossil "github.com/dburkart/fossil/api"
```

Comment:
Just for my own curiosity... what's the difference between `fossil "github.com/dburkart/fossil/api"` and `"github.com/dburkart/fossil/api"`?

Reply:
That was a govet change I believe, but the it is an alias of the package name. Since `api` is a common and nondescript name it is best to alias them to something that is self documenting ie

```go
c, _ := fossil.Client()
```
vs 
```go
c, _ := api.Client()
```

Reply:
Oh I did not notice that... It is generally best practice to keep the package name the same as the folder. I now have no clue if this is because of that or not.